### PR TITLE
Add a pref for webxr glwindow anaglyphic red-cyan stereo

### PR DESCRIPTION
**Pull request by asajeffrey**
Allows us to test out 3D content using old-skule glasses.

---

- [x] `./mach build -d` does not report any errors
- [x] `./mach test-tidy` does not report any errors
- [x] These changes do not require tests because it's back end work

### CODE REVIEW

**Comment by asajeffrey** [comment]
Shouldn't land until https://github.com/servo/webxr/pull/166 does.

File: components/config/prefs.rs [hunk 1]
```diff
@@ -297,11 +297,15 @@ mod gen {
...
-                    glwindow: bool,
+                    glwindow: {
+                        #[serde(default)]
+                        enabled: bool,
+                    },
                     hands: {
                         #[serde(default)]
                         enabled: bool,
+                        #[serde(rename = "dom.webxr.glwindow.red-cyan")]
+                        red_cyan: bool,
```
Comment (jdm) [inline components/config/prefs.rs:308]: Ooh, red-cyan hands!

Comment (asajeffrey) [reply components/config/prefs.rs]: Thank you rebase conflict resolution. Fixed now.

**Comment by asajeffrey** [comment]
The webxr change has landed. r? @Manishearth

**Comment by jdm** [comment]
Taskcluster is red.

**Comment by asajeffrey** [comment]
Yeah, I don't understand why, it appears not to have applied the `fixup` commit. I'll rebase and force-push, and see if that fixes things.

File: Cargo.toml [hunk 2]
```diff
@@ -34,3 +34,7 @@ spirv_cross = { git = "https://github.com/servo/spirv_cross", branch = "wgpu-ser
 surfman-chains = { git = "https://github.com/asajeffrey/surfman-chains" }
 surfman = { git = "https://github.com/servo/surfman" }
 
+[patch."https://github.com/servo/webxr"]
+webxr = { git = "https://github.com/asajeffrey/webxr", branch = "glwindow-red-cyan" }
+webxr-api = { git = "https://github.com/asajeffrey/webxr", branch = "glwindow-red-cyan" }
```
Comment (jdm) [inline Cargo.toml:39]: This should be removed now, yes?

**Review by jdm — CHANGES_REQUESTED** [review]

File: Cargo.toml (hunk 2 above)
Comment (asajeffrey) [reply Cargo.toml]: Done.

File: Cargo.lock [hunk 3]
```diff
@@ -6428,7 +6428,7 @@ dependencies = [
 [[package]]
 name = "webxr"
 version = "0.0.1"
-source = "git+https://github.com/servo/webxr#eae68436697131122504b035746daa3a157b36b4"
+source = "git+https://github.com/asajeffrey/webxr?branch=glwindow-red-cyan#2091631b6ef33404ddbfda251b4aeb1a9bcb02c0"
```
Comment (Manishearth) [inline Cargo.lock:6431]: No patch please

**Review by Manishearth — APPROVED** [review]

File: Cargo.lock (hunk 3 above)
Comment (asajeffrey) [reply Cargo.lock]: Done.

**Comment by jdm** [comment]
@bors-servo retry
* #25513

**Pull request merged**